pythonbits/bb.py

```diff
@@ -396,14 +404,27 @@ def _render_audio_codec(self):
                 c = c.replace('AC3', 'AC-3')
                 return c
 
+        if audio_track['codec'] == 'DTS-HD':
+            if ('format_profile' in audio_track and
+                    audio_track['format_profile'] == 'X / MA / Core'):
+                return 'DTS:X'
+            return 'DTS-HD'
```

Comment:
If I'm not mistaken, these `return`s will never be reached, due to line 397/403/405

Reply:
These changes have been tested and are working as expected. Both DTS:X and DTS-HD as well as for the TrueHD block. What you describe makes more sense to eliminate any unnecessary code. I'll update it when I can.